### PR TITLE
HPCC-15110 Load Graph XGMML from query dll instead of Dali

### PR DESCRIPTION
**Pull request by jakesmith**
Signed-off-by: Jake Smith jake.smith@lexisnexis.com

### CODE REVIEW

**Comment by jakesmith** [comment]
@richardkchapman - please review

File: thorlcr/graph/thgraph.cpp [hunk 1]
```diff
@@ -2488,6 +2488,14 @@ CJobBase::CJobBase(const char *_graphName) : graphName(_graphName)
     jobSlaveChannelNum.allocateN(querySlaves()); // filled when channels are added.
     for (unsigned s=0; s<querySlaves(); s++)
         jobSlaveChannelNum[s] = NotFound;
+    StringBuffer wuXML;
+    if (!getEmbeddedWorkUnitXML(querySo, wuXML))
+        throwUnexpected();
```
Comment (richardkchapman) [inline thorlcr/graph/thgraph.cpp]: Should these throw more descriptive error messages? Or do you consider it sufficiently unlikely that we don't care?

Comment (jakesmith) [inline thorlcr/graph/thgraph.cpp]: getEmbeddedWorkUnitXML() already throws an exception if !loadCurrentExecutable(), so probably impossible to return NULL.

Comment (jakesmith) [inline thorlcr/graph/thgraph.cpp]: Actually I lied (looking at wrong method) - I will change to a more meaningful exception.

**Comment by jakesmith** [comment]
@richardkchapman - have changed exception on getEmbeddedWorkUnitXML() failure to something meaningful and squashed.

**Pull request merged**